frontend/components/atoms/Label.js

```diff
@@ -0,0 +1,10 @@
+import styled from 'styled-components';
+import { SystemLabel } from './SystemComponents';
+
+const Label = styled(SystemLabel)``;
```

Comment:
I think `styled(SystemLabel).attrs({ fontFamily: ... })` could work and allows these props to be overrideable too?